cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -370,25 +369,116 @@ private void addCimCharacteristics(Network network) {
     }
 
     private void convertACLineSegmentsToLines(Context context, Set<String> delayedBoundaryNodes) {
+        Map<String, PropertyBags> lineContainers = new HashMap<>();
         Iterator<PropertyBag> k = cgmes.acLineSegments().iterator();
         while (k.hasNext()) {
             PropertyBag line = k.next();
             if (LOG.isDebugEnabled()) {
                 LOG.debug(line.tabulateLocals("ACLineSegment"));
             }
             ACLineSegmentConversion c = new ACLineSegmentConversion(line, context);
+            if (c.valid()) {
+                String lineContainerId = line.getId("Line");
+                if (lineContainerId != null) {
+                    lineContainers.computeIfAbsent(lineContainerId, l -> new PropertyBags()).add(line);
```

Comment:
After the latest changes, I still see no reason to delay the conversion of ACLS contained in a cim:Line. We just have to create the fictitious voltage levels and/or substations associated to them.